src/components/LoginPage/Login.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+
+function Login(){
+
+  return(
+  <div>
+      <div className='box'>
+        <h2 className='login-h2'>Login</h2>
+        <form >
+          <div className="input-box">
+            <input type='text' placeholder="Enter Username" required=""></input>
+
+          </div>
+          <div className="input-box">
+            <input type="password" placeholder="Enter Password" required=""></input>
+
+          </div>
+            <input className='submit-btn' type="submit" name="" value="submit"></input>
```

Comment:
Did you mean to not capitalize "submit"?

Reply:
Yes, as far as I knew. There is no need to capitalize. Are you talking on the type="submit"?